From 56aeb6f86f2674a72e73e83e686627c1fdd57e43 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Tue, 22 Jul 2014 14:38:55 -0400
Subject: [virt] kvm/x86: remove unused last_kernel_ns variable

Message-id: <1406040016-3289-141-git-send-email-pbonzini@redhat.com>
Patchwork-id: 86219
O-Subject: [RHEL7 PATCH v2 140/221] KVM: x86: remove unused last_kernel_ns variable
Bugzilla: 1116936
RH-Acked-by: Radim Krcmar <rkrcmar@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Marcelo Tosatti <mtosatti@redhat.com>

Remove unused last_kernel_ns variable.

Signed-off-by: Marcelo Tosatti <mtosatti@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
(cherry picked from commit 4f34d683e52271197e1ee17b7095e8ba27761ba6)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/include/asm/kvm_host.h b/arch/x86/include/asm/kvm_host.h
index d0d22a2..dcff727 100644
--- a/arch/x86/include/asm/kvm_host.h
+++ b/arch/x86/include/asm/kvm_host.h
@@ -444,7 +444,6 @@ struct kvm_vcpu_arch {
  } st;
 
  u64 last_guest_tsc;
- u64 last_kernel_ns;
  u64 last_host_tsc;
  u64 tsc_offset_adjustment;
  u64 this_tsc_nsec;
diff --git a/arch/x86/kvm/x86.c b/arch/x86/kvm/x86.c
index 520b9b3..88d67e8 100644
--- a/arch/x86/kvm/x86.c
+++ b/arch/x86/kvm/x86.c
@@ -1584,7 +1584,6 @@ static int kvm_guest_time_update(struct kvm_vcpu *v)
  /* With all the info we got, fill in the values */
  vcpu->hv_clock.tsc_timestamp = tsc_timestamp;
  vcpu->hv_clock.system_time = kernel_ns + v->kvm->arch.kvmclock_offset;
- vcpu->last_kernel_ns = kernel_ns;
  vcpu->last_guest_tsc = tsc_timestamp;
 
  /*
-- 
1.7.1